rules/windows/registry/registry_set/registry_set_add_load_service_in_safe_mode.yml

```diff
@@ -1,4 +1,4 @@
-title: Registry Persitence via Service in Safe Mode
+title: Registry Persistence via Service in Safe Mode
```

Comment:
https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_specification.md#modified-optional
modified has to be updated